platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -683,8 +684,8 @@ public class SwirldsPlatform implements Platform, PlatformWithDeprecatedMethods,
             // SwirldStateManager will get a copy of the state loaded, that copy will become stateCons.
             // The original state will be saved in the SignedStateMgr and will be deleted when it becomes old
 
-            preConsensusEventHandler = components.add(PlatformConstructor.preConsensusEventHandler(
-                    platformContext.getMetrics(), threadManager, selfId, swirldStateManager, consensusMetrics));
+            preConsensusEventHandler = components.add(
+                    new PreConsensusEventHandler(metrics, threadManager, selfId, swirldStateManager, consensusMetrics));
```

Comment:
Was this change from using `PlatformConstructor` to just calling `new` intentional, or due to a merge conflict?